frameworks: native: pick egl/gles libraries from the correct path also in the 64 bit case.

krnlyng opened the pull request:
[frameworks] native: pick egl/gles libraries from the correct path also in the 64 bit case. JB#53238

Also use the new way of exporting patches.

New patch is:  0004-hybris-Pick-GLES-shared-objects-from-appropriate-pat.patch

Commit:
frameworks: native: export patches with -N --zero-commit --full-index --no-signature.

[frameworks] native: export patches with -N --zero-commit --full-index --no-signature. JB#53238

mlehtima reviewed (approved):
Looks good. Although modifying the original patch might be cleaner.

sledges commented on art/0001-hybris-Silence-some-build-errors-to-let-the-systemim.patch:
```diff
@@ -1,4 +1,4 @@
-From 1a27bd48255a96853e49d25c82a0fe47dd477f83 Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```
Please first cleanup the patches so it makes finding/reverting the change easier:) Cf. https://github.com/mer-hybris/hybris-patches/pull/11/commits

krnlyng replied:
@sledges done

Commit:
frameworks: native: pick egl/gles libraries from the correct path also in the 64 bit case.

[frameworks] native: pick egl/gles libraries from the correct path also in the 64 bit case. JB#53238
